packages/frontend/src/pages/Flows/index.tsx

```diff
@@ -72,6 +106,14 @@ export default function Flows(): React.ReactElement {
     [],
   )
 
+  if (loading) {
+    return <CircularProgress sx={{ display: 'block', margin: '20px auto' }} />
```

Comment:
Hmm this one means the header and the search input won't show until the `loading` is back to false eh (which is ok-ish as we're using `loading` only for first load), but I think it makes more sense in the old place where it won't hide the input and page title. Any particular reason we move it here?

Reply:
wah this one was tricky to me... so like if i put `loading` in the original place, an issue occurs: when the user have flows, the `EmptyFlowsTemplate` will pop out for a split second before showing the user's flows. 

Then, to fix this, I tried stacking `!loading` check for the `EmptyFlowsTemplate` and another issue will occur: when the user does not have flows, the input and page title will pop out for a split second before showing the `EmptyFlowsTemplate`.

Hence, I thought the best fix was just to have the `loading` check right at the top which will work regardless of whether the user has flows or not. And hence, this compromises the UI you mentioned, would this be ok for you?

Reply:
oh ok makes sense then since we dont want to show those on EmpyFlow